coala_quickstart/Strings.py

```diff
@@ -33,3 +33,13 @@
              "all its contents but exclude your .git directory and all "
              ".o files. To do this, simply give `src/` for the first "
              "question and `.git/**,**/*.o` for the second question.")
+
+# Start ignoring LineLengthBear
+bear_docs_url = "https://github.com/coala-analyzer/bear-docs/blob/master/README.rst#supported-languages"
+# Stop ignoring
+bear_help = ("A coala bear is a plugin that contains the actual algorithm. "
+             "It have its own native code or use an external linter. It "
+             "may be language specific or language independent. This "
+             "makes coala completely modularized and extensible. "
+             "Currently there are over 75 bears for over 35 languages.\n\n"
```

Comment:
so we avoid maintenance, you should maybe say that we have support for many many languages and provide some examples (like C) imo^^